database.html

```diff
@@ -0,0 +1,112 @@
+<!doctype html>
+<html lang="en">
+  <head>
+	<!-- Required meta tags -->
+	<meta charset="utf-8">
+	<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
+
+	<!-- Bootstrap CSS -->
+	<link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/4.1.3/css/bootstrap.min.css">
+	
+	<!-- jQuery library, then Popper JS, then JavaScript -->
+	<script src="https://ajax.googleapis.com/ajax/libs/jquery/3.3.1/jquery.min.js"></script>
+	<script src="https://cdnjs.cloudflare.com/ajax/libs/popper.js/1.14.3/umd/popper.min.js"></script>
+	<script src="https://maxcdn.bootstrapcdn.com/bootstrap/4.1.3/js/bootstrap.min.js"></script>
+	
+	<title>MyMood Database Interface</title>
+  </head>
+  
+  <body>
+	
+	<!-- Navigation Bar -->
+	<nav class="navbar navbar-expand-sm navbar-dark bg-primary" id="mainNav">
+		<div class="container-fluid">
+			<a class="navbar-brand" href="database.html">MyMood</a>
+			<ul class="navbar-nav ml-auto">
+				<li class="nav-item active">
+					<a class="nav-link" href="database.html">Database
+						<span class="sr-only">(current)</span>
+					</a>
+				</li>
+				<li class="nav-item">
+					<a class="nav-link" href="help.html">Help</a>
+				</li>
+				<li class="nav-item">
+					<a class="nav-link" href="aboutus.html">About Us</a>
+				</li>
+				<!-- TODO: Light/Dark Theme Options -->
+				<li class="nav-item dropdown">
+					<a class="nav-link dropdown-toggle" href="#" id="navbardrop" data-toggle="dropdown">
+						Light Theme
+					</a>
+					<div class="dropdown-menu">
+						<a class="dropdown-item" href="#">Dark Theme</a>
+					</div>
+				</li>
+				<li class="nav-item">
+					<a class="nav-link" href="logout.html">Logout</a>
+				</li>
+			</ul>
+		</div>
+	</nav>
+	<br>
+	
+	<!-- Database Table -->
+	<div class="container table-responsive">
+		<h1>MyMood Database Table</h1>
+		<!-- Search Bar 
+		TODO: Add more filtering options other than text -->
+		<input class="form-control" id="searchInput" type="text" placeholder="Search..">
+		<br>
+		<table class="table table-sm table-striped table-bordered table-hover id="session_table">

```

Comment:
Mismatched quotes? Attribute "id" in string "table table-sm table-striped table-bordered table-hover id=".

Reply:
looks like it should be
<table class="table table-sm table-striped table-bordered table-hover" id="session_table">